.travis.yml

```diff
@@ -3,12 +3,10 @@ sudo: false
 git:
   depth: 1
 cache:
-  directories:
-    - $HOME/.npm
-    - node_modules
+  yarn: true
 matrix:
   include:
-    - node_js: "11"
+    - node_js: "12"
```

Comment:
Node 11 has stopped the support:
https://github.com/nodejs/Release/tree/ae09392b7e64fa1a947e27556427c3b19d487fc2#release-schedule